packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -318,7 +318,9 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.sign_out,
           () async {
+            // ignore: use_build_context_synchronously
             if (await _confirmLogout(context) == true) {
+              // ignore: use_build_context_synchronously
```

Comment:
For this guy, you can also start your code with `final NavigatorState navigatorState = Navigator.of(context)`, and then `navigatorState.pop()`.